marlowe-playground-client/src/Simulation.purs

```diff
@@ -327,6 +321,24 @@ handleGistAction settings LoadGist = do
 
   toEither x NotAsked = x
 
+loadGist ::
+  forall m.
+  MonadAff m =>
+  MonadEffect m =>
+  Gist ->
+  HalogenM State Action ChildSlots Void m (Either String Unit)
+loadGist gist =
+  runExceptT do
+    currentContract <- noteT "Source not found in gist." $ preview (_Just <<< gistFileContent <<< _Just) (currentSimulationMarloweGistFile gist)
+    let
+      oldContract = preview (_Just <<< gistFileContent <<< _Just) (oldSimulationMarloweGistFile gist)
```

Comment:
Am I right in thinking that:

`preview (_Just <<< gistFileContent <<< _Just)`

...is:

`view (_Just <<< gistFileContent)`

?

Reply:
yes you are!